torchci/lib/JobClassifierUtil.ts

```diff
@@ -141,6 +141,29 @@ export function getConclusionChar(conclusion?: string): string {
   }
 }
 
+export function getConclusionSeverityForSorting(conclusion?: string): number {
+  // Returns a severity level for the conclusion. 
+  // Used to sort jobs by severity
+  switch (conclusion) {
+    case JobStatus.Success:
+      return 0;
+    case JobStatus.Skipped:
+      return 1;
+    case JobStatus.Neutral:
+      return 2;
+    case JobStatus.Cancelled:
+      return 3;
+    case JobStatus.Pending:
+      return 4;
+    case undefined:
```

Comment:
lol why is this 5/what corresponds to this case?

Reply:
Heh, I don't know if that case can ever really happen, but the existing code covers it so I followed suite.  

I chose to highlight this weird case in the UI for now so that if it ever occurs it gets ranked relatively high and gets attended to. The logic:

- Failure is stuff folks will definitely pay attention to
- Pending is less interesting but tends to get eyeballed if someone is waiting
- Right between those two seemed like a good spot for `undefined`